traitsui/qt4/check_list_editor.py

```diff
@@ -266,7 +267,7 @@ def update_object(self, event=None):
         """Handles the user changing the contents of the edit control."""
         try:
             value = str(self.control.text())
-            value = eval(value)
+            value = literal_eval(value)
```

Comment:
I'm not 100% sure about this one because I don't have much experience with the editor.